### PR TITLE
OSDOCS-2028: Release notes for new wildcard subject

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-2028

Preview: https://deploy-preview-32969--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-security-wildcard-subjects-headless-services

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -57,6 +57,21 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-security"]
 === Security and compliance
 
+[id="ocp-4-8-security-wildcard-subjects-headless-services"]
+==== Wildcard subject for service serving certificates for headless services
+
+Generating a service service certificate for headless services now includes a wildcard subject in the format of `*.<service.name>.<service.namespace>.svc`. This allows for TLS-protected connections to individual stateful set pods without having to manually generate certificates for these pods.
```
Comment (bobfuru) [inline release_notes/ocp-4-8-release-notes.adoc:63]: "service service certificate" -- should this be "service serving certificate" or perhaps just "service certificate"?

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -57,6 +57,21 @@ This release adds improvements related to the following components and concepts.
...
+
+[IMPORTANT]
+====
+Because the generated certificates contain wildcard subjects for headless services, you must not use the service CA if your client must differentiate between individual pods. In this case, do the following:
+
+* Generate individual TLS certificates using a different CA.
+* Do not accept the service CA as a trusted CA for connections that are directed to individual pods and must not be impersonated by other pods. These connections must be configured to trust the CA that was used to generate the individual TLS certificates.
```
Comment (bobfuru) [inline release_notes/ocp-4-8-release-notes.adoc:70]: - I often find "do not" to be easier to understand than "you must not", which might also help here because "must" appears again in the same sentence. 🤷 

- `s/using/by using/` (for clarification)

- Also, can you rephrase the lead-in sentence? ISG doesn't like "do the following (and "Do the following...Do not accept" is a bit of a contradiction).

  **following** _n_
  Do not use as a noun except to refer to a group of followers, adherents, or partisans. For example, do not
write “Do the following” or “The functions include the following” as lead-in wording to a list. Use
“following” only as an adjective, as in “Complete the following steps” or “The adapter has the following
functions.”

**Review by bobfuru — COMMENTED** [review]
A few minor nits but otherwise LGTM!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc:63]: You're right, thanks for catching that! It's a mouthful :)

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc:70]: @bobfuru Yeah I struggled with that lead-in to the list. I didn't like "do the following" but I did see it a few places in the repo and couldn't think of anything better. But I'm definitely up for finding a better alternative - let me know if you can think of any suggestions! I'll look at this fresh again tomorrow and see if I get a stroke of genius ;)

Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc]: I shortened to just "In this case" for now, but will think on it again

Comment (bobfuru) [reply release_notes/ocp-4-8-release-notes.adoc]: I think "In this case" works great in this case! 😁

**Review by mtrmac — COMMENTED** [review]
LGTM

**Comment by barleyer** [comment]
LGTM
